syntax = "proto3";

package mars;

service MarsService {
  rpc GetStatus(StatusRequest) returns (StatusResponse);
}

message StatusRequest {}

message StatusResponse {
  string status = 1;
}
